flasc/energy_ratio/energy_ratio.py

```diff
@@ -299,6 +300,8 @@ def get_energy_ratio(
         N=1,
         percentiles=[5.0, 95.0],
         return_detailed_output=False,
+        block_bootstrapping = False, 
```

Comment:
Remove spaces around equal sign for consistency

Reply:
Done!